chain/events/filter/index.go

```diff
@@ -270,6 +272,7 @@ func (ei *EventIndex) PrefillFilter(ctx context.Context, f *EventFilter) error {
 			event.reverted,
 			event_entry.flags,
 			event_entry.key,
+			event_entry.codec,
```

Comment:
We should be matching on codec + value, but that's not critical right now because we only support CBOR.

Reply:
I think you mean Raw?

Reply:
I think you mean Raw?